reference/5.1/Microsoft.PowerShell.Utility/Out-Printer.md

```diff
@@ -76,8 +83,8 @@ Accept wildcard characters: False
 ```
 
 ### -Name
-Specifies the alternate printer.
-The parameter name *Name* is optional.
+
+Specifies the alternate printer. The parameter name **Name** is optional.
```

Comment:
Not sure alternate is the right word here.  Maybe "specific" like previous examples that also used **Name**.

Reply:
fixed